### PR TITLE
Add rosbag2csv

### PR DESCRIPTION
**Pull request by furushchev**
add rosbag csv exporter since rqt_bag no longer supports csv export.

### CODE REVIEW

File: jsk_topic_tools/scripts/bag2csv.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    """
+    stream: StringIO
+    msg: message
+    """
+    try:
+        for s in type(msg).__slots__:
+            val = msg.__getattribute__(s)
+            message_type_to_csv(stream, val, ".".join([parent_content_name,s]))
+    except:
+        stream.write(", " + parent_content_name)
```
Comment (garaemon) [inline jsk_topic_tools/scripts/bag2csv.py]: I think no space is required after `,`

**Comment by garaemon** [comment]
`rostopic echo -p` is enough?

```
$ rostopic echo hoge -p
%time,field.data
1423155837204827070,hoge
1423155838204751014,hoge
1423155839204807043,hoge
```

**Comment by garaemon** [comment]
rostopic also can read bag file with `-b` option.
```
Usage: rostopic echo [options] /topic

Options:
  -h, --help            show this help message and exit
  -b BAGFILE, --bag=BAGFILE
                        echo messages from .bag file
  -p                    echo in a plotting friendly format
  --filter=FILTER-EXPRESSION
                        Python expression to filter messages that are printed.
                        Expression can use Python builtins as well as m (the
                        message) and topic (the topic name).
  --nostr               exclude string fields
  --noarr               exclude arrays
  -c, --clear           clear screen before printing next message
  -a, --all             display all message in bag, only valid with -b option
  -n COUNT              number of messages to echo
  --offset              display time as offsets from current time (in seconds)

```

**Comment by k-okada** [comment]
Please contribute to rqt_bag, we need to escape from infinite loop ->
http://answers.ros.org/question/9102/how-to-extract-data-from-bag/?answer=63877#post-id-63877

**Comment by garaemon** [comment]
I don't think rqt_bag should have this feature. rostopic is enough

✉︎ Ryohei Ueda




On Thu, Feb 5, 2015 at 5:10 PM -0800, "Kei Okada" <notifications@github.com> wrote:












Please contribute to rqt_bag, we need to escape from infinite loop ->

http://answers.ros.org/question/9102/how-to-extract-data-from-bag/?answer=63877#post-id-63877

**Comment by k-okada** [comment]
if rostopic can handle this , that it much much better,

--
◉ Kei Okada

**Pull request merged**